x/ccv/provider/types/keys.go

```diff
@@ -166,6 +166,10 @@ const (
 	// per validator per consumer chain
 	ConsumerCommissionRatePrefix
 
+	// ConsumerCommissionRateDenomPrefix is the byte prefix for storing the
```

Comment:
Rename `ConsumerCommissionRateDenomPrefix` to `MinimumPowerInTopNBytePrefix`.

Reply:
oops :D